diff --git a/chain/dev/config.toml b/chain/dev/config.toml
index 434e79f5..8ae7aeac 100644
--- a/chain/dev/config.toml
+++ b/chain/dev/config.toml
@@ -14,9 +14,6 @@ grandpa = ""
 sync = ""
 digest = ""
 
-[init]
-genesis = "./chain/dev/genesis.json"
-
 [account]
 key = ""
 unlock = ""
diff --git a/chain/dev/defaults.go b/chain/dev/defaults.go
index 88193c2d..636a83b3 100644
--- a/chain/dev/defaults.go
+++ b/chain/dev/defaults.go
@@ -18,7 +18,7 @@
 	// DefaultID is the chain ID
 	DefaultID = string("dev")
 	// DefaultConfig is the toml configuration path
-	DefaultConfig = string("./chain/dev/config.toml")
+	DefaultConfig = string("@out@/share/dev/config.toml")
 	// DefaultBasePath is the node base directory path
 	DefaultBasePath = string("~/.gossamer/dev")
 
@@ -39,7 +39,7 @@
 	// InitConfig
 
 	// DefaultGenesis is the default genesis configuration path
-	DefaultGenesis = string("./chain/dev/genesis.json")
+	DefaultGenesis = string("@out@/share/dev/genesis.json")
 
 	// AccountConfig
 
diff --git a/chain/gssmr/config.toml b/chain/gssmr/config.toml
index 5165b96f..58c9acc6 100644
--- a/chain/gssmr/config.toml
+++ b/chain/gssmr/config.toml
@@ -14,9 +14,6 @@ grandpa = ""
 sync = ""
 digest = ""
 
-[init]
-genesis = "./chain/gssmr/genesis.json"
-
 [account]
 key = ""
 unlock = ""
diff --git a/chain/gssmr/defaults.go b/chain/gssmr/defaults.go
index 0295be7a..c42b49b6 100644
--- a/chain/gssmr/defaults.go
+++ b/chain/gssmr/defaults.go
@@ -20,7 +20,7 @@
 	// DefaultID Default chain ID
 	DefaultID = string("gssmr")
 	// DefaultConfig Default toml configuration path
-	DefaultConfig = string("./chain/gssmr/config.toml")
+	DefaultConfig = string("@out@/share/gssmr/config.toml")
 	// DefaultBasePath Default node base directory path
 	DefaultBasePath = string("~/.gossamer/gssmr")
 
@@ -41,7 +41,7 @@
 	// InitConfig
 
 	// DefaultGenesis is the default genesis configuration path
-	DefaultGenesis = string("./chain/gssmr/genesis.json")
+	DefaultGenesis = string("@out@/share/gssmr/genesis.json")
 
 	// AccountConfig
 
diff --git a/chain/kusama/config.toml b/chain/kusama/config.toml
index 865fd01f..b6ece58e 100644
--- a/chain/kusama/config.toml
+++ b/chain/kusama/config.toml
@@ -14,9 +14,6 @@ grandpa = ""
 sync = ""
 digest = ""
 
-[init]
-genesis = "./chain/kusama/genesis.json"
-
 [account]
 key = ""
 unlock = ""
diff --git a/chain/kusama/defaults.go b/chain/kusama/defaults.go
index 57183b18..b99b8209 100644
--- a/chain/kusama/defaults.go
+++ b/chain/kusama/defaults.go
@@ -18,7 +18,7 @@
 	// DefaultID Default chain ID
 	DefaultID = string("ksmcc3")
 	// DefaultConfig Default toml configuration path
-	DefaultConfig = string("./chain/kusama/config.toml")
+	DefaultConfig = string("@out@/share/kusama/config.toml")
 	// DefaultBasePath Default node base directory path
 	DefaultBasePath = string("~/.gossamer/kusama")
 
@@ -39,7 +39,7 @@
 	// InitConfig
 
 	// DefaultGenesis is the default genesis configuration path
-	DefaultGenesis = string("./chain/kusama/genesis.json")
+	DefaultGenesis = string("@out@/share/kusama/genesis.json")
 
 	// AccountConfig
 
diff --git a/chain/polkadot/config.toml b/chain/polkadot/config.toml
index 26ae4ba7..bbfb218c 100644
--- a/chain/polkadot/config.toml
+++ b/chain/polkadot/config.toml
@@ -14,9 +14,6 @@ grandpa = ""
 sync = ""
 digest = ""
 
-[init]
-genesis = "./chain/polkadot/genesis.json"
-
 [account]
 key = ""
 unlock = ""
diff --git a/chain/polkadot/defaults.go b/chain/polkadot/defaults.go
index 2a756488..1b9a3200 100644
--- a/chain/polkadot/defaults.go
+++ b/chain/polkadot/defaults.go
@@ -18,7 +18,7 @@
 	// DefaultID Default chain ID
 	DefaultID = string("polkadot")
 	// DefaultConfig Default toml configuration path
-	DefaultConfig = string("./chain/polkadot/config.toml")
+	DefaultConfig = string("@out@/share/polkadot/config.toml")
 	// DefaultBasePath Default node base directory path
 	DefaultBasePath = string("~/.gossamer/polkadot")
 
@@ -36,7 +36,7 @@
 	// InitConfig
 
 	// DefaultGenesis is the default genesis configuration path
-	DefaultGenesis = string("./chain/polkadot/genesis.json")
+	DefaultGenesis = string("@out@/share/polkadot/genesis.json")
 
 	// AccountConfig
 
