diff --git a/cmake/dependencies.cmake b/cmake/dependencies.cmake
index 6fd79e1..36c6871 100644
--- a/cmake/dependencies.cmake
+++ b/cmake/dependencies.cmake
@@ -37,7 +37,7 @@ find_package(OpenSSL REQUIRED)
 
 # https://developers.google.com/protocol-buffers/
 hunter_add_package(Protobuf)
-find_package(Protobuf CONFIG REQUIRED)
+find_package(Protobuf REQUIRED)
 
 # http://rapidjson.org
 hunter_add_package(RapidJSON)
