diff --git a/cmake/install.cmake b/cmake/install.cmake
index 13fff58f..66cb6401 100644
--- a/cmake/install.cmake
+++ b/cmake/install.cmake
@@ -3,7 +3,7 @@ function(kagome_install target)
     install(TARGETS ${target} EXPORT kagomeTargets
         LIBRARY       DESTINATION ${CMAKE_INSTALL_LIBDIR}/kagome
         ARCHIVE       DESTINATION ${CMAKE_INSTALL_LIBDIR}/kagome
-        RUNTIME       DESTINATION ${CMAKE_INSTALL_BINDIR}/kagome
+        RUNTIME       DESTINATION ${CMAKE_INSTALL_BINDIR}
         INCLUDES      DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/kagome
         PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/kagome
         FRAMEWORK     DESTINATION ${CMAKE_INSTALL_PREFIX}/kagome
