diff --git a/core/runtime/common/CMakeLists.txt b/core/runtime/common/CMakeLists.txt
index 4220f3b7..659a4687 100644
--- a/core/runtime/common/CMakeLists.txt
+++ b/core/runtime/common/CMakeLists.txt
@@ -25,7 +25,7 @@ add_library(uncompress_if_needed
     uncompress_code_if_needed.cpp
     )
 target_link_libraries(uncompress_if_needed
-    zstd::libzstd_static
+    zstd
     )
 kagome_install(uncompress_if_needed)
 
